h2o-automl/src/main/java/ai/h2o/automl/EventLogEntry.java

```diff
@@ -0,0 +1,192 @@
+package ai.h2o.automl;
```

Comment:
– renamed from UserFeedbackEvent.
– added support for event metadata (1 per event): cf. setNamedValue()